tests/test_closing.py

```diff
@@ -4042,3 +4042,19 @@ def test_closing_no_anysegwit_retry(node_factory, bitcoind):
 
     oldaddr = l1.rpc.newaddr()['bech32']
     l1.rpc.close(l2.info['id'], destination=oldaddr)
+
+
+def test_closing_ignore_fee_limits(node_factory, bitcoind, executor):
+    """Don't use ignore-fee-limits on mutual close: LDK takes us to the cleaners if we do!"""
+    l1, l2 = node_factory.line_graph(2, opts=[{'may_reconnect': True,
+                                               'ignore-fee-limits': True},
+                                              {'may_reconnect': True}])
+
+    # l2's feerates go up.  A lot!
+    l2.set_feerates((100000, 100000, 100000, 100000))
+    l2.restart()
+    l1.rpc.connect(l2.info['id'], 'localhost', l2.port)
+
+    # This fails to negotiate.
+    fut = executor.submit(l1.rpc.close, l2.info['id'])
```

Comment:
```suggestion
    _ = executor.submit(l1.rpc.close, l2.info['id'])
```

I think we need just to fix the CI

```
cc common/utils.c
ld tools/check-bolt
tests/test_closing.py:4059:5: F841 local variable 'fut' is assigned to but never used
make: *** [Makefile:535: check-python-flake8] Error 1
rm external/build-x86_64-linux-gnu/libwally-core-build/src/secp256k1/libsecp256k1.la
```

Reply:
Ah, I can just omit the assignment completely, thanks!